test/e2e/operator_test.go

```diff
@@ -91,7 +92,31 @@ func TestVersionReporting(t *testing.T) {
 		t.Errorf("failed to get deployment %s/%s: %v", deployment.Namespace, deployment.Name, err)
 	}
 
-	patch := []byte(`{"spec": {"template": {"spec": {"containers": [{"name":"dns-operator","env":[{"name":"RELEASE_VERSION","value":"0.0.1-test"}]}]}}}}`)
+	numContainers := len(deployment.Spec.Template.Spec.Containers)
+	if numContainers != 1 {
+		t.Errorf("expected operator deployment to contain 1 container but found %d", numContainers)
+	}
```

Comment:
This doesn't hurt but shouldn't be necessary—the API says there must be at least one container.